pkg/shardddl/pessimism/doc.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 PingCAP, Inc.
```

Comment:
This `doc.go` mainly talk about the situation that combining tables that are in two workers. Should we add more explannation about the situation of combing tables that are in one worker?
For example, combining
worker1: t_1, worker1: t_2, worker2: t_1, worker3, t_2 => tidb:t

Reply:
Coordination between multiple tables in one DM-worker is the responsibility of the DM-worker itself. For the full explanation, see https://pingcap.com/blog-cn/tidb-ecosystem-tools-3/#sharding-ddl.